src/es-module-shims.js

```diff
@@ -269,11 +269,11 @@ function resolveDeps (load, seen) {
           if (!(blobUrl = depLoad.s)) {
             blobUrl = depLoad.s = createBlob(`export function u$_(m){${
               depLoad.a[1].map(
-                name => name === 'default' ? `d$_=m.default` : `${name}=m.${name}`
+                expt => expt.n === 'default' ? `d$_=m.default` : `${expt.n}=m.${expt.n}`
               ).join(',')
             }}${
-              depLoad.a[1].map(name =>
-                name === 'default' ? `let d$_;export{d$_ as default}` : `export let ${name}`
+              depLoad.a[1].map(expt =>
+                expt.n === 'default' ? `let d$_;export{d$_ as default}` : `export let ${expt.n}`
```

Comment:
I wonder if we can update this to properly support string exports now / non identifier exports in general rather than just default?

One way would be to number all identifiers (`e$1`, `e$2`, ...) and then export a bag - `export { e$1 as ${source.slice(expt.s, expt.e)}, ... }` kind of thing. Let me know if that makes sense?

Reply:
Done. We already have the local name, so I don't think we need to generate identifiers.

The two cyclic test cases now create:

```js
export function u$_(m){f=m.f,d$_=m.default}let f;let d$_;export {f as f,d$_ as default}
//# sourceURL=http://localhost:8088/test/fixtures/test-circular1.js?cycle

export function u$_(m){b=m.b,c=m.d,e=m['e f'],d$_=m.default}let b;let c;let e;let d$_;export {b as b,c as d,e as 'e f',d$_ as default}
//# sourceURL=http://localhost:8088/test/fixtures/test-self-import.js?cycle
```